seed/challenges/00-getting-started/getting-started.json

```diff
@@ -602,6 +677,63 @@
           "https://gitter.im/apps"
         ]
       ],
+      "TitleIt": "Crea un account GitHub e iscriviti alle nostre Chat Rooms",
+      "DescriptionIt": [
+        [
+          "//i.imgur.com/EAR7Lvh.jpg",
+          "Uno screenshot di una delle nostre chat rooms in Gitter.",
+          "Prima di iniziare a scrivere codice, entra nelle chat di freeCodeCamp. Potrai venire qui in ogni momento della giornata per chiacchierare, fare domande, o trovare un altro camper per programmare in coppia. Prima di tutto avrai bisogno di un account GitHub.",
+          ""
+        ],
+        [
+          "//i.imgur.com/n6GeSEm.gif",
+          "Una gif che invita a cliccare sul link qui sotto per andare su GitHub. Compila i campi necessari e fare clic su Invia. Quindi controlla la tua casella di posta elettronica, riceverai una e-mail da GitHub per verificare il tuo account.",
+          "Clicca sul pulsante \" Apri link in una nuova scheda \" qui sotto per aprire GitHub. Crea un account compilando il modulo. Assicurati di utilizzare un indirizzo email reale - GitHub lo terrà privato. Poi controlla la tua casella di posta, riceverai un'e-mail da GitHub. Aprirlo e clicca il pulsante \"verificare l'indirizzo e-mail \" al suo interno <br> <div class=\"small\">. Nota: Se disponi già di un account di GitHub, è possibile saltare questo passaggio cliccando \"Apri link in una nuova scheda \", chiudendo poi la scheda appena aperta, quindi facendo clic su \"Vai al mio prossimo passo \". Abbiamo rimosso il pulsante \"Salta il passaggio \" perché molte persone lo premerebbero ripetutamente, saltando questi importanti passaggi. </ div> ",
+          "https://github.com/join"
+        ],
+        [
+          "//i.imgur.com/hFqAEr8.gif",
+          "Una gif che mostra come cliccare l'immagine del profilo nell'angolo in alto a destra di GitHub. Carica una foto di te o si continuerai ad utilizzare la pixel art generata automaticamente. Poi, compila i campi del modulo rimanenti e fai clic su Invia.",
+          "Clicca sulla pixel art nell'angolo in alto a destra di GitHub, quindi seleziona impostazioni. Carica una foto di te stesso. Una foto del viso è la scelta migliore. Gli altri campers ti vedranno così nelle nostre chat rooms, quindi cerca di apparire al meglio . È possibile aggiungere la tua città e il tuo nome, se vuoi.",
```

Comment:
"È **anche** possibile aggiungere la tua città e il tuo nome". The "se vuoi" seems excessively redundant in italian.